x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_latest_version/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { NOT_AVAILABLE_LABEL } from '../../../../../../common/i18n';
+import { AgentName } from '../../../../../../typings/es_schemas/ui/fields/agent';
+
+export function AgentLatestVersion({
+  agentName,
+  latestVersion,
+  timedOut,
+}: {
+  agentName: AgentName;
+  latestVersion?: string;
+  timedOut?: boolean;
+}) {
+  if (!timedOut) {
+    return latestVersion ? <>{latestVersion}</> : <>{NOT_AVAILABLE_LABEL}</>;
+  }
+
+  return (
+    <EuiToolTip
+      content={i18n.translate(
+        'xpack.apm.agentExplorer.agentLatestVersion.airGappedMessage',
+        {
+          defaultMessage:
+            'Kibana server needs public access to internet in order to get latest {agentName} agent released version.',
```

Comment:
`internet` might be too broad here. What about showing the domain of the bucket URL? Or just a message that is more specific about needing access to the bucket?

Reply:
What about: `The latest agent versions could not be fetched from the repository. Please contact your administrator to check the server logs.` - and then log a warning server side with the URL from the repository?

Reply:
Addressed in [3130a75](https://github.com/elastic/kibana/pull/153643/commits/3130a7506f9572fe9a633ffaeeacbf7c9b9937aa).